src/index.template.ejs

```diff
@@ -1,6 +1,5 @@
 <!DOCTYPE html>
 <html lang="en">
-  <% const data = require('../site-data.json') %>
```

Comment:
The external data you can pass into the template via Webpack config using the [data](https://github.com/webdiscus/html-bundler-webpack-plugin#loader-option-data) option of the HtmlBundlerPlugin